composer/src/main/kotlin/com/gojuno/composer/Main.kt

```diff
@@ -111,15 +113,25 @@ private fun runAllTests(args: Args, testPackage: TestPackage.Valid, testRunner:
                     val installTimeout = Pair(args.installTimeoutSeconds, TimeUnit.SECONDS)
                     val installAppApk = device.installApk(pathToApk = args.appApkPath, timeout = installTimeout)
                     val installTestApk = device.installApk(pathToApk = args.testApkPath, timeout = installTimeout)
+                    val coverageDir = "/storage/emulated/0/app_coverage/${testPackage.value}"
```

Comment:
Shouldn't we use `EXTERNAL_STORAGE` environment variable instead of hardcoding external storage directory?